debug_utils/sierra-emu/tests/common/mod.rs

```diff
@@ -0,0 +1,89 @@
+use sierra_emu::Value;
+use starknet_crypto::Felt;
+
+/// Convert a Value to a felt.
+pub fn jitvalue_to_felt(value: &Value) -> Vec<Felt> {
```

Comment:
Its not a `jitvalue`, but a sierra emu value:
```suggestion
/// Convert a Value to a felt.
pub fn value_to_felt(value: &Value) -> Vec<Felt> {
```

Reply:
My bad

Reply:
Done